app/models/plum_schema.rb

```diff
@@ -1,328 +1,346 @@
 # frozen_string_literal: true
 module PlumSchema
   extend ActiveSupport::Concern
-  included do
-    # Hyrax
-    attribute :depositor
-    attribute :title
-    attribute :label
-    attribute :visibility
+  def imported_schema
```

Comment:
Can we make an issue for that? I'm not sure the neatest way to bisect them. The only reason this exists is so that importing metadata won't wipe out all the non-imported metadata (IE `source_metadata_identifier`)